src/popup/vault/groupings.component.ts

```diff
@@ -57,6 +57,7 @@ export class GroupingsComponent extends BaseGroupingsComponent implements OnInit
     showLeftHeader = true;
     searchPending = false;
     searchTypeSearch = false;
+    deletedCount: number = 0;
```

Comment:
You don't have to specify type if you are initing it.

```suggestion
    deletedCount = 0;
```

Reply:
Fixed